Automattic/jetpack
# Contact Form: Update child blocks icons

Fixes #28015

#### Changes proposed in this Pull Request:
* Updates the Contact Form child blocks icons as proposed on #28015 

#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Create a post and check if the Form child blocks icons were updated accordingly
